concourse/pipeline.yml

```diff
@@ -1507,7 +1507,7 @@ jobs:
           args:
           - -exc
           - |
-            mvn --settings iac-service-source/.travis.settings.xml install -Ddockerfile.skip -f iac-service-source/pom.xml
+            mvn --settings iac-service-source/.maven.settings.xml install -Ddockerfile.skip -f iac-service-source/pom.xml
```

Comment:
I might be looking in the wrong place, but I couldn't spot this in the root of the repo?

Reply:
Nope you're just a good review :D

Reply:
https://github.com/ONSdigital/iac-service/compare/rename-travis-settings-to-maven-settings?expand=1